test/powershell/engine/Jobs.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe 'Basic Job Tests' -Tags 'CI' {
+
+    $job = Start-Job {1}
+
+    It 'Test job creation' {
+        $job | should not be $null
+    }
+
+    It 'Test job State' {
+        Wait-Job $job -Timeout 5
+        $job.JobStateInfo.State -eq 'Completed' | should be $true
+    }
+
+    It 'Job output test' {
+        Receive-Job $job -wait | should be 1
+    }
+}
```

Comment:
Converting the repository to not be missing line endings would be appreciated.
But I would still encourage you to also discourage people from adding files that are missing `EOL` at `EOF` because unless your script automatically commits the line changes as the user who added the line, you'll be distorting authorship for those lines.